module/roi/abstract_roi.hpp

```diff
@@ -2,6 +2,67 @@
 
 #include <fmt/core.h>
 
+#include <iostream>
+#include <opencv2/opencv.hpp>
+
 namespace abstract_roi {
```

Comment:
Add a new line after namespace start and before namespace end would be better:

```suggestion
namespace abstract_roi {

```